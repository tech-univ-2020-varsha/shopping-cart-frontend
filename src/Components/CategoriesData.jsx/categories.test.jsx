import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import Categories from './categories';

describe('The Categories component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Categories data={[]} filter="All" total={1} setTotal={() => {}} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
