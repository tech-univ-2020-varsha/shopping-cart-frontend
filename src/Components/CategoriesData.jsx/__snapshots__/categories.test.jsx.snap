// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Categories component should render correctly 1`] = `
<DocumentFragment>
  <div>
    <div>
      All
    </div>
    <div />
  </div>
</DocumentFragment>
`;
